packages/fxa-settings/src/pages/SigninConfirmed/index.tsx

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import Ready from '../../components/Ready';
+
+type SigninConfirmedProps = {
+  continueHandler?: Function;
+  serviceName?: string;
+};
+
+const SigninConfirmed = ({
+  continueHandler,
+  serviceName,
+}: SigninConfirmedProps & RouteComponentProps) => {
+  const viewName = 'signin-confirmed';
+
```

Comment:
Just realized: this is missing logging the page view event with test which we'll then add to the spreadsheet as implemented. This is the only thing blocking me from r+!!

Reply:
oh shoot I'll add it in, thank you!

Reply:
I looked into this, and the page view event is actually logged in the `Ready` component, which might be a bad pattern. (Like it's nice that we don't have to duplicate the code across each page, but it's not nice that it seems to a reader that it isn't being logged!) I'll add a test in which shows that it is being logged, but we can also make a ticket to move that logging out of the shared component if desirable

Reply:
...oh wait, there already is a page view event log test! Or am I misunderstanding?

Reply:
Just said this in Slack but yeah, I missed it since we're doing it in the Ready component and I missed the test for it somehow. After you next push I'll do one more quick pass and r+. 😎